products/bigquery/terraform.yaml

```diff
@@ -91,6 +91,11 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     properties:
       datasetId: !ruby/object:Overrides::Terraform::PropertyOverride
         ignore_read: true
+      role: !ruby/object:Overrides::Terraform::PropertyOverride
+        diff_suppress_func: 'resourceBigQueryDatasetAccessRoleDiffSuppress'
+        custom_expand: "templates/terraform/custom_expand/bigquery_access_role.go.erb"
```

Comment:
You need the custom expander because that's what gets used to find the item in the list, right? If so can you add a comment explaining as much? It took me a second to figure it out.

Reply:
👍